odoo/addons/base/models/res_users.py

```diff
@@ -325,7 +325,7 @@ def _get_default_image(self):
                                  compute='_compute_accesses_count', compute_sudo=True)
     groups_count = fields.Integer('# Groups', help='Number of groups that apply to the current user',
                                   compute='_compute_accesses_count', compute_sudo=True)
-    image_1920 = fields.Image(related='partner_id.image_1920', inherited=True, readonly=False, default=_get_default_image)
+    image_1920 = fields.Image(related='partner_id.image_1920', inherited=True, readonly=False)
```

Comment:
maybe this line can be removed now that it doesn't have specific default?
Field should already come from partner inherit/mixin